app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -24,13 +25,14 @@ if [ -f /etc/os-release ] ; then
   . /etc/os-release
 fi
 
-create_host_curl() {
-  curl --request POST <%= foreman_server_url %>/api/hosts \
+register_host() {
+  curl --request POST <%= foreman_server_url %>/register \
```

Comment:
Should we use `curl --fail` instead?

Reply:
But then you won't get nice errors.

```
register_host(){
curl --request POST http://foreman.example.com:3000/register --user admin:changeme -d '{ "host": { "name": "invalid/name"}}' -H 'Content-Type: application/json'
}

register_host | bash
ERROR: standard_error
Validation failed: Name hostname can contain only ...

register_host --fail | bash
curl: (22) The requested URL returned error: 500 Internal Server Error
```

From user perspective it's better the option without `--fail`